wandb/sdk_py27/lib/module.py

```diff
@@ -15,11 +15,11 @@ def set_global(
 ):
     if run:
         wandb.run = run
-    if config:
+    if config is not None:
         wandb.config = config
     if log:
         wandb.log = log
-    if summary:
+    if summary is not None:
```

Comment:
When are config and summary None?

Reply:
This is to differentiate between `None` (default) and empty dict (in disabled mode case).

Reply:
but an empty dict is an empty wandb_config.Config() object, which will evaluate to true.